api/platformpb/json/platformpb.json

```diff
@@ -0,0 +1,190 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "swagger": "2.0",
+  "info": {
+    "title": "PMM Platform API",
+    "version": "v0"
+  },
+  "paths": {
+    "/v0/Platform/Connect": {
```

Comment:
I don't think we should expose this endpoint. Once user calls it the result will be the following:
- new inventory will be added to Portal organization inventory list
- the SSO params are returned
- but PMM Server doesn't apply SSO Params -> out of sync state of PMM Server and Portal Inventory.

Reply:
That is not true. This won't return sso params. This calls Platform API, stores SSO and reloads grafana.

Reply:
This call returns empty response and 200 on success, different code and error json on failure.